src/__main__.py

```diff
@@ -37,33 +37,49 @@
     "graysexual": [54, 242, 255, 242, 54],
 }
 
+# A dictionary of all the available stats
+stats = {
+    "os": distribution() or system() or 'N/A',
+    "arch": architecture() or 'N/A',
+    "pkgs": packages() or 'N/A',
+    "kernel": kernel() or system() or 'N/A',
+    "uptime": str(timedelta(seconds=clock_gettime(CLOCK_BOOTTIME))).split('.', 1)[0]
+}
```

Comment:
This is neat! One thing I want to check is that, does lambda have to be applied for all the functions, i.e. kernel() and system(), or just having it in the current position will apply for all functions?

Definite performance increase with these changes though, when running custom stats:
`python __main__.py -s os  0.06s user 0.01s system 99% cpu 0.066 total`
`python __main__.py -s pkgs  0.12s user 0.04s system 74% cpu 0.214 total`

Reply:
This is neat! One thing I want to check is that, does lambda have to be applied for all the functions

`lambda` defines a function, i.e. doing this
```py
foo = lambda: "foo"
```
is roughly equivalent to doing
```py
def foo():
    return "foo"
```

When you have multiple functions with or conditionals, the line evaluates roughly to
```py
def function():
    return function1() or function2()
```

This can be seen with the following snippet:
```py
>>> def foo(bar):
...     print(bar)
... 
>>> baz = lambda: foo(1) or foo(2)
>>> baz()
1
2
```

Reply:
Great